package.json

```diff
@@ -13,16 +13,23 @@
     "@types/react": "^16.9.52",
     "@types/react-dom": "^16.9.0",
     "@types/react-redux": "^7.1.9",
+    "@types/react-router": "^5.1.8",
+    "@types/react-router-dom": "^5.1.5",
     "firebase": "^7.23.0",
     "node-sass": "^4.14.1",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
     "react-redux": "^7.2.1",
     "react-redux-firebase": "^3.7.0",
     "react-router": "^5.2.0",
+    "react-router-dom": "^5.2.0",
     "react-scripts": "3.4.3",
     "redux-firestore": "^0.14.0",
-    "typescript": "~4.0.3"
+    "typescript": "~4.0.3",
+    "redux-thunk": "^2.3.0"
+  },
+  "devDependencies": {
+    "redux-devtools-extension": "^2.13.8"
```

Comment:
Why is this the first `dev` depedency? As you might have noticed in my other comments I very much appreciate uniform solutions :D
If you introduce the `dev` part here I would also suggest to move all `@types` and other unneccessary packages for the production build here